pkg/statedb/api_handler.go

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package statedb
+
+import (
+	"net/http"
+
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/runtime/middleware"
+
+	. "github.com/cilium/cilium/api/v1/server/restapi/statedb"
+	restapi "github.com/cilium/cilium/api/v1/server/restapi/statedb"
+)
+
+func createHandler(db DB) restapi.GetStatedbDumpHandler {
+	return &dumphandler{db}
+}
+
+// REST API handler for the StateDB, handles the JSON Dumping.
```

Comment:
Hey, let's add comment here that this is available as "cilium statedb dump" and that this output is included in the sysdump.